tests/phpunit/tests/template.php

```diff
@@ -494,6 +494,45 @@ public function test_locate_template_uses_current_theme() {
 		$this->assertSame( $new_theme->get_stylesheet_directory() . '/index.php', locate_template( $template_names ), 'Incorrect index template found in theme after switch.' );
 	}
 
+	/**
+	 * Tests that `locate_template()` uses the current theme even after switching the theme.
+	 *
+	 * @dataProvider data_locate_template_only_loads_theme_files
+	 *
+	 * @ticket 58905
+	 *
+	 * @covers ::locate_template
```

Comment:
```suggestion
	 * @ticket 58905
	 *
	 * @covers ::locate_template
	 *
	 * @dataProvider data_locate_template_only_loads_theme_files
```

Correct the annotation order.